LandSandBoat/server
# [lua] Dynamis Valk Boss Mobskill Audit

**_I affirm:_**

## What does this pull request do?
Adds missing mobskills for bosses in dynamis valk.
Also added minor immunities and mechanics as seen in captures. More in-depth mechanics will be PRd when we get more captures but for now the skills are implemented and used. They are no longer regular mobs that auto attack and do nothing.

See https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI/edit?gid=57955395#gid=57955395 for more info regarding type of damage.

Dynamis Valkurm - mobskill id gathering for all nightmare mobs
https://youtu.be/aQZxaOS8eF8
https://1drv.ms/u/s!AlLEVQXhZeaHgRmEXha5w9t872mc?e=ezMbFm
Dynamis Valkurm - Nightmare Funguar mobskill testing
https://youtu.be/ksumDSG5g3s
https://1drv.ms/u/s!AlLEVQXhZeaHgRWO5Uu-g3KN7YTP?e=FJwkD2
Dynamis Valkurm - Gouging Branch mobskill testing
https://youtu.be/MKlqoam6-M4
https://1drv.ms/u/s!AlLEVQXhZeaHgRT6znXrTUj2I5IG?e=7FJU3y 
Cirrate Christelle
https://www.youtube.com/watch?v=QHcGtTR_xQg
Nant'ina
https://www.youtube.com/watch?v=TaRrYdYjJJw



## Steps to test these changes
!gotoid 16936961
See that it uses miasmic_breath, fragrant_breath, putrid_breath, vampiric_lash
!gotoid 16937051
See that it uses gouging branch
!gotoid 16937077
See that it uses Blow x9 -> Uppercut x3 -> Attractant
!gotoid 16937100
See that its very fast and spams mephitic spore
